server/controllers/PartyController.js

```diff
@@ -0,0 +1,32 @@
+import parties from '../models/parties';
+import { logoUrlUploaded } from '../middleware/imageUpload';
+
+class PartyController {
+  static createParty(req, res, next) {
+    const {
+      name, hqAddress, logoUrl, acronym,
```

Comment:
logoUrl is unused in this module. Any idea why?

Reply:
@akinwale-habibullah I forgot to remove it. It's unused because it is parsed as req.file instead of req.body.

Reply:
Ok. Great you've removed it now.